src/models/stores/ui.ts

```diff
@@ -59,6 +59,7 @@ export const UIModel = types
   .volatile(self => ({
     defaultLeftNavExpanded: false,
     standalone: false,
+    errorContent: undefined as React.FC<any> | undefined,
```

Comment:
I wonder if this should be called `errorComponent` to make the typing more clear.